boards/nordic/nrf5340dk/CMakeLists.txt

```diff
@@ -2,14 +2,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 if ((CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP OR CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP_NS)
-    AND CONFIG_BOARD_ENABLE_CPUNET)
-zephyr_library()
-zephyr_library_sources(nrf5340_cpunet_reset.c)
-
-if (CONFIG_BUILD_WITH_TFM)
+    AND CONFIG_BUILD_WITH_TFM)
   zephyr_library_include_directories(
     $<TARGET_PROPERTY:tfm,TFM_BINARY_DIR>/api_ns/interface/include
   )
```

Comment:
It should be safe to remove this as well. If I understand 88bb3ae498426d83b560acdd7470fd378e693d90 correctly, this was only introduced because at that time `soc_secure.h` [was including `tfm_ioctl_api.h`](https://github.com/zephyrproject-rtos/zephyr/blob/88bb3ae498426d83b560acdd7470fd378e693d90/soc/arm/nordic_nrf/nrf53/soc_secure.h#L12) , so that path was missing for `subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/debug.h` that was included by `nrf5340_cpunet_reset.c`. But after soc_secure refactoring in 6a14a0233b7eb6d66d4380992d2e292470e05a72, this became unneeded.
Same applies to `bl5340_dvk` and `nrf5340_audio_dk`.

Reply:
added one more commit to delete these includes